osquery/tables/templates/default.cpp.in

```diff
@@ -32,7 +32,7 @@ class {{class_name}} {
 class {{table_name_cc}}TablePlugin : public TablePlugin {
 public:
   TableName name = "{{table_name}}";
-  TableColumns columns = {
+  const TableColumns columns = {
```

Comment:
We should add a note into `tables.h` about having to update this file when updating the `TablePlugin` class.